pages/portfolio.html

```diff
@@ -1,5 +1,6 @@
 <!DOCTYPE html>
 <html lang="en">
+<a href="https://adadevelopersacademy.org/">Click here to visit Ada's website!</a>
```

Comment:
Since this is the main part of your page, it doesn't belong at the top of this HTML page. We should put main content in a main tag: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/main

So you'd have your html tag, then head tag, then your body tag. 

Inside your body tag, you can put your main tag and then the anchor tag, like this:
```html
<body>
      <main>
            <a href="https://adadevelopersacademy.org/">Click here to visit Ada's website!</a>
     </main>
</body>

Reply:
Right now when I go to your portfolio page, there aren't navigation links to let me go to your home page or your about page. I have to use my browser's back button to get back home. 

Consider adding a nav tag to the body tag with an unordered list of links to other pages on your site -- like you have in your home page (index.html)